docs/baking.md

```diff
@@ -155,6 +155,26 @@ sudo -u tezos tezos-client bootstrapped
 By default `tezos-baking-<network>.service` will use the `baker` alias for the
 key that will be used for baking and endorsing.
 
+### Setting the Liquidity Baking toggle vote option
+
+Since `013-PtJakart`, the `--liquidity-baking-toggle-vote` command line option for
+`tezos-baker` is now mandatory. In our systemd services, it is set to `pass` by
+default. You can change it as desired in the service config file at
+`/etc/defaults/tezos-baking-<network>`.
+
+If the baking service is running already, changing this options requires a restart.
+You can do so manually by running:
+```bash
+sudo systemctl restart tezos-baking-<network>.service
+```
```

Comment:
I just realized that one piece of documentation is missing: in the `docs/tests/voting.md` file there is an example of the service config file to edit containing:
```
DATA_DIR="/var/lib/tezos/.tezos-client"
NODE_RPC_ENDPOINT="http://localhost:8732"
BAKER_ADDRESS_ALIAS="baker"
CUSTOM_NODE_CONFIG="<paste your path to the node config here>"
RESET_ON_STOP=""
```
which is missing the `LIQUIDITY_BAKING_TOGGLE_VOTE` line